flexmeasures/api/v1/implementations.py

```diff
@@ -201,7 +201,7 @@ def collect_connection_and_value_groups(
         # Get the power values
         # TODO: fill NaN for non-existing values
         power_bdf_dict: Dict[str, tb.BeliefsDataFrame] = Power.collect(
-            generic_asset_names=asset_names,
+            old_sensor_names=asset_names,
```

Comment:
I think the `collect` function on old sensor data classes will be entirely replaced by a `search_beliefs` function on the TimedBelief class. It should accept sensor ids rather than names.